### ISSUE TITLE
Parsing week numbers using d3.time.format

### ISSUE THREAD

**Issue by mchinosi** [mbostock/d3#1914]
Hi.
I have a problem parsing weeks.
I'm using d3 v3.4.8.
I have dates in a format like `2014-19` - but the format is not a problem, since I have the same problem using `2014(19)`, `2014 19` and so on...
Is it possible to parse such string using `$Y-$W` or `$Y-$U`? e.g.
```javascript
d3.time.format('%Y-%W').parse('2014-19');
```
Because I obtain as a result always the date corresponding to the 1st of January 2014
```javascript
Date {Wed Jan 01 2014 00:00:00 GMT+0100}
```
I'm also parsing months, years, days and it works well. Only weeks are not working.

Thank you
Michele

**Comment by mbostock** [comment]
New behavior:

```
> d3.time.format('%Y-%W').parse('2014-19');
Mon May 12 2014 00:00:00 GMT-0700 (PDT)
> d3.time.format('%Y-%U').parse('2014-19');
Sun May 11 2014 00:00:00 GMT-0700 (PDT)
```

**Comment by mbostock** [comment]
Fix staged in #2591 for 3.5.7.
